src/neuroglancer/async_computation/request.ts

```diff
@@ -34,7 +36,7 @@ function returnWorker(worker: Worker) {
 }
 
 function getNewWorker(): Worker {
-  const worker = new Worker('async_computation.bundle.js');
+  const worker = new Worker(bundleUrl);
```

Comment:
Since this is now just a constant you could just inline it.

Reply:
Fixed